src/components/Tooltip/index.jsx

```diff
@@ -2,18 +2,33 @@ import React from 'react';
 import { string, node } from 'prop-types';
 import MuiTooltip from '@material-ui/core/Tooltip';
 
+/**
+ * Tooltip accessible by hovering mouse or touching mobile screen.
+ */
 function Tooltip({ title, children }) {
+  /**
+   * Forward the properties and ref of the tooltip to the children components
+   * so that MuiTooltip has control over the wrapped components' props.

```

Comment:
Why do we want the tooltip to have control over the wrapped component. Is the `div` that we're rendering not enough?

Reply:
I initially thought we would need this per the document https://material-ui.com/components/tooltips/#custom-child-element , however, simply using the `div` for wrapping any children works just as well.